powerstore/provider_test.go

```diff
@@ -19,6 +19,9 @@ var testProviderFactory map[string]func() (tfprotov6.ProviderServer, error)
 var endpoint = os.Getenv("POWERSTORE_ENDPOINT")
 var username = os.Getenv("POWERSTORE_USERNAME")
 var password = os.Getenv("POWERSTORE_PASSWORD")
+var hostID = os.Getenv("HOST_ID")
+var hostGroupID = os.Getenv("HOST_GROUP_ID")
+var volumeGroupID = os.Getenv("VOLUME_GROUP_ID")
```

Comment:
Passing IDs creates following problem:
1. one needs to first create them via some interface and let these resource be present there with no use but only for the sake of testing
creating them first in pre processing and delete them post processing will be the best choice.
Just like we are creating resource under testing and deleting them

Reply:
As per conversation, I am marking it as resolved.
Reason : Not convenient possible to create and delete as per need
these hostID , hostGroupID and volumeGroupID cannot be create on the way, but need a complete independent flow where permission also need to be asked manually from other team.